### PR TITLE
hubspot: Add purpose annotations

### PR DESCRIPTION
**Pull request by jcharum**
Also format.

### CODE REVIEW

File: hubspot/plugin.py [hunk 1]
```diff
@@ -622,55 +627,55 @@ def hubspot_update_contacts(
...
-recent_deal_close_date: Date last deal associated with contact was closed-won.
-closedate: Date the contact became a customer.
-
-Default Properties of Type Boolean:
-hs_email_bad_address: The email address associated with this contact is invalid.
-hs_is_contact: Is a contact, has not been deleted and is not a visitor.
-hs_is_unworked: Contact has not been assigned or has not been engaged after last owner assignment/re-assignment.
-hs_sequences_is_enrolled: A yes/no field that indicates whether the contact is currently in a Sequence.
-"""
+    contact_updates is a dict mapping contact id to a list of tuples with the property names to update, and their new values.
```
Comment (bozhi) [inline hubspot/plugin.py:630]: Let's not make this change because these are tokens that the LLM would have to use.

File: hubspot/plugin.py [hunk 2]
```diff
@@ -1084,51 +1108,51 @@ def hubspot_update_companies(
...
-hs_num_open_deals: The number of open deals associated with this company.
-total_revenue: The total amount of closed won deals.
-num_associated_contacts: The number of contacts associated with this company.
-num_associated_deals: The number of deals associated with this company.
-
-Default Properties of Type Boolean:
-hs_is_target_account: Identifies whether this company is being marketed and sold to as part of your account-based strategy.
-is_public: Indicates if the company is publicly traded.
-"""
+    company_updates is a dict mapping company id to a list of tuples with the property names to update, and their new values.
```
Comment (bozhi) [inline hubspot/plugin.py:1111]: ditto regarding LLM tokens

File: hubspot/plugin.py [hunk 3]
```diff
@@ -1561,44 +1589,44 @@ def hubspot_update_deals(
...
-closedate: Date the deal was closed, set automatically by HubSpot.
-createdate: Date the deal was created, set automatically by HubSpot.
-hs_closed_won_date: Returns the closedate if the deal is closed won.
-hs_lastmodifieddate: The most recent timestamp of any property update for this deal.
-
-Default Properties of Type Boolean:
-hs_is_closed: True if the deal was won or lost.
-hs_is_closed_won: True if the deal is in the closed-won state.
-"""
+    deal_updates is a dict mapping deal id to a list of tuples with the property names to update, and their new values.
```
Comment (bozhi) [inline hubspot/plugin.py:1592]: same

File: hubspot/plugin.py (hunk 1 above)
Comment (jcharum) [reply hubspot/plugin.py:630]: Did you observe this?  My belief is that it does not change the content of what's passed to the LLM (the result of [`doc_string()`](https://github.com/d8e-ai/alda/blob/e6463769a510ba4b2d1a2e1d41bf74ca89123b1d/v3/actions/plugin_action.py#L60-L61)), as we [use `get_docstring`](https://github.com/d8e-ai/alda/blob/e6463769a510ba4b2d1a2e1d41bf74ca89123b1d/v3/actions/plugin.py#L540).  [`clean` defaults to `True`](https://docs.python.org/3.11/library/ast.html#ast.get_docstring), so I think all leading whitespace should be removed.

Comment (bozhi) [reply hubspot/plugin.py:630]: Ah ok. I assumed that is was being passed through.

**Review by bozhi — APPROVED** [review]

**Comment by jcharum** [comment]
@bozhi: Merged; can you re-import?

**Pull request merged**